### PR TITLE
Snippeting chapter9

### PR DESCRIPTION
**Pull request by Casteran**
Start to add a few snippets to chapter9 (primitive recursive functions).

-  There is a quotation of a definition from the Ackermann library, and the same problem will happen with the Standard Library. It looks difficult to add snippets in such libraries. I chose (for the time being) to leave the Coqsrc blocks. 

- Just for trying, I put a commented proof of extEqual_ex1 (P. 165 of the pdf, library ordinals/MoreAck/PrimRecExamples.v)  with comments inside the proof. I put a few (* .no-outs *) comments because alectryon displays the sub-goal after the bullets.

### CODE REVIEW

**Review by Zimmi48 — COMMENTED** [review]
Remembering to put all the `(* .no-out *)` can be bothering. We can do better by setting this parameter for whole blocks of code with:

```coq
(*|
.. coq:: no-out
|*)
```

I'll push a commit doing that.

File: doc/chapter-primrec.tex [hunk 1]
```diff
@@ -876,154 +743,121 @@ \subsection{Proof steps}
...
 
-\begin{Coqsrc}
-H'_F : forall (alpha : E0) (n : nat), 
-     F_ alpha (S n) <= H'_ (Phi0 alpha) (S n) 
-\end{Coqsrc}
+\input{movies/snippets/F_alpha/HprimeF}
 
 
-Our proof of this lemma is not trivial at all. One of it sub-goals is the following one:
+Our proof of this lemma is not trivial at all, it uses some properties of the Ketonen-Solovay's toolkit. We advise the reader to explore this proof, with the help of an ide or softwre like alectryon.
```
Comment (Zimmi48) [inline doc/chapter-primrec.tex:770]: softwre -> software

**Comment by Casteran** [comment]
I add a little (improvable) text on the use of alectryon in the doc (pages 8 and 11 of the pdf)
@Zimmi48 Do you think we may merge into master ?

Do you think we should mention alectryon in the "installation" paragraph of the README ?

**Comment by Zimmi48** [comment]
> I add a little (improvable) text on the use of alectryon in the doc (pages 8 and 11 of the pdf)

Looks good!

> Do you think we may merge into master ?

Yes :)

> Do you think we should mention alectryon in the "installation" paragraph of the README ?

It already mentions Alectryon:

>Building the PDF documentation also requires Alectryon 1.2 and SerAPI. See doc/movies/Readme.md for details.

Let me know if you think this isn't sufficient.

**Comment by Casteran** [comment]
Ok !
I let you merge ?
Thé, I will send a message to Clement with a link to the pdf generated by CI.

Envoyé de mon iPhone

**Pull request merged**